lasio/las.py

```diff
@@ -704,6 +704,43 @@ def set_data_from_df(self, df, **kwargs):
                 str(name) for name in df.columns.values
             ]
         self.set_data(df_values, **kwargs)
+    
+    def stack_curves(self, curve_list=None, stub=None, sort_curves=True):
+        """Stack multi-channel curve data to a numpy 2D ndarray. Provide a 
+        list of curves or a stub name (prefix shared by all curves that will
+        be stacked). If `stub` and `curve_list` are both supplied, `stub`
+        is used.
+        
+        Keyword Arguments:
+            stub (str): Supply the first several characters of the channel
+                set to be stacked. 
+            curve_list (list): Or a list of the curves to be stacked
```

Comment:
Clarify whether the list must contain mnemonics, or whether CurveItems can be used too.

Reply:
Good point. For the moment I just intended this method to take curve mnemonics which I've now clarified in the doc string. Maybe I can add CurveItems as an option in a future PR